docs/guide/design-principles/scale-out.md

```diff
@@ -49,6 +51,8 @@ A primary advantage of the cloud is elastic scaling &mdash; the ability to use a
 
 **Consider scaling for redundancy.** Scaling out can improve your application's reliability. For example, consider scaling out across multiple [availability zones](/azure/reliability/availability-zones-overview), such as by using zone-redundant services. This approach can improve your application's throughput as well as provide resiliency if one zone experiences an outage.
 
+**Model and optimize your system's scalability.** You can use e.g. [Universal Scalability Law](http://www.perfdynamics.com/Manifesto/USLscalability.html) (USL) to model and to optimize scalability of your system. USL is an extension of [Amdahl's law](https://en.wikipedia.org/wiki/Amdahl%27s_law) and quantifies scalability based on parameters such as contention and coherency.  Contention refers to delay due to waiting or queueing for shared resources. Coherence refers to delay for data to become consistent. For example, having a high contention indicates sequential processing that could be parallelized, while having a high coherency suggests excessive dependencies among processes, prompting you to minimize interactions. Also, with help of USL, you can, in advance, calculate the maximum effective capacity of your system: scaling up your system beyond that point is a waste.
```

Comment:
```suggestion
**Model and optimize your system's scalability.** You can use model your system using an approach such as [Amdahl's law](https://wikipedia.org/wiki/Amdahl's_law). Quantify scalability based on parameters such as contention and coherency.  Contention refers to delay due to waiting or queueing for shared resources. Coherence refers to delay for data to become consistent. For example, having a high contention indicates sequential processing that could be parallelized, while having a high coherency suggests excessive dependencies among processes, prompting you to minimize interactions. During workload design, you can calculate the maximum effective capacity of your system to avoid providing [more supply than demand which leads to waste](/azure/well-architected/cost-optimization/optimize-scaling-costs#optimize-demand-and-supply).
```